examples/nirf/rfsa-rfsg-NR-analysis-only-modacc.py

```diff
@@ -0,0 +1,467 @@
+  C:\Program Files (x86)\National Instruments\RFmx\NR\Documentation\rfmxnrcvi.chm
+
+Getting Started:
+
+To run this example, install "RFmx NR", "RFSA" and "RFSG" on the server machine:
+  https://www.ni.com/en-us/support/downloads/software-products/download.rfmx-nr.html
+  https://www.ni.com/en-us/support/downloads/drivers/download.ni-rfsa.html
+  https://www.ni.com/en-us/support/downloads/drivers/download.ni-rfsg.html
+
+
+For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC
+Client" wiki page:
+  https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+
+Refer to the NI-RFmxNR gRPC Wiki for the latest C Function Reference:
+  https://github.com/ni/grpc-device/wiki/NI-RFmxNR-C-Function-Reference
+
+Running from command line:
+
+Server machine's IP address, port number, and physical channel name can be passed as separate
+command line arguments.
+  > python rfsa-rfsg-NR-analysis-only-modacc.py <server_address> <port_number> <resource_name>
+If they are not passed in as command line arguments, then by default the server address will be
+"localhost:31763", with "SimulatedDevice" as the resource name.
+"""
+
+import sys
+
+import grpc
+import nidevice_pb2 as nidevice_grpc
+import nirfmxinstr_pb2 as nirfmxinstr_types
+import nirfmxnr_pb2 as nirfmxnr_types
+import nirfmxnr_pb2_grpc as grpc_nirfmxnr
+import nirfsa_pb2 as nirfsa_types
+import nirfsa_pb2_grpc as grpc_nirfsa
+import nirfsg_pb2 as nirfsg_types
+import nirfsg_pb2_grpc as grpc_nirfsg
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+RFMXSESSION_NAME = "RFmxNRSession"
+RFSASESSION_NAME = "RfsaSession"
+RFSGSESSION_NAME = "RfsgSession"
+
+# Resource name and options for a simulated 5663 client.
+RESOURCE = "5840_1"
+RFMXOPTIONS = "AnalysisOnly=1"
+RFSAOPTIONS = "DriverSetup=SFPSessionAccess:1"
+RFSGOPTIONS = ""
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+if len(sys.argv) >= 4:
+    RESOURCE = sys.argv[3]
+    OPTIONS = ""
+# Create a gRPC channel + client.
+# Use a larger buffer for the messages as we are transferring the raw data from instrument.
+# In this example up to 1.6M samples
+options = [("grpc.max_receive_message_length", 2**21)]
+channel = grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}", options=options)
+rfmxclient = grpc_nirfmxnr.NiRFmxNRStub(channel)
+rfsaclient = grpc_nirfsa.NiRFSAStub(channel)
+rfsgclient = grpc_nirfsg.NiRFSGStub(channel)
+rfsgsession = None
+rfmxsession = None
+rfsasession = None
+
+
+def raise_if_initialization_error(response):
+    """Raise an exception if an error was returned from Initialize."""
+    if response.status < 0:
+        raise RuntimeError(f"Error: {response.error_message or response.status}")
+    if response.status > 0:
+        sys.stderr.write(f"Warning: {response.error_message or response.status}\n")
+    return response
+
+
+def raise_if_error(response):

```

Comment:
Can you replace this `raise_if_error` with `check_for_warning` like the other examples have? You can see it [here](https://github.com/ni/grpc-device/blob/main/examples/nirfmxgsm/orfs.py#L78) in the recently added RFmx GSM example.